elastic/kibana
# [Synthetics] fix script_type for monitor management telemetry

## Summary

Fixes Monitor Management telemetry by adding the appropriate `script_type` to the request body for browser monitors.

### Testing
The easiest way to test this is to add a console log [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/synthetics/server/lib/telemetry/sender.ts#L181)
```
console.warn('ndjson', ndjson);
```
Testing inline monitors
--
1. Create an inline browser monitor in monitor management
2. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `inline` (You'll see two events each time)
3. Edit the same inline browser monitor. 
4. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `inline`
5. Delete the same inline browser monitor
6. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `inline`

Testing recorder monitors
1. Create a recorder browser monitor in monitor management by uploading this file: https://github.com/elastic/synthetics/blob/main/examples/inline/short.js
2. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `recorder` (You'll see two events each time)
3. Edit the same recorder browser monitor. 
4. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `recorder`
5. Delete the same recorder browser monitor
6. Navigate to Kibana logs. Wait for the telemetry event to fire after about 30 seconds. Confirm that `script_type` is `recorder`
Delete any items that are not applicable to this PR.
